tests/cypress/e2e/features/single_object_annotation.js

```diff
@@ -228,6 +235,23 @@ context('Single object annotation mode', { scrollBehavior: false }, () => {
             cy.get('.cvat-player-next-button').click();
             checkFrameNum(2);
 
+            // Change label and draw an object,
+            const anotherLabelName = 'points_label';
+
```

Comment:
I think its better to be placed in separate test case. It may be a bit slower, but from general perspective we shouldnt mix testing two different issues together. The problem may arise in controls or after switching workspaces. If this test fails, we wont know what exactly happened.
(+dont need to enable navigate empty again)